api/apps/api/src/modules/projects/dto/legacy-project-import.dto.ts

```diff
@@ -14,6 +14,7 @@ export class StartLegacyProjectImportBodyDto {
   })
   @IsString()
   projectName!: string;
+  solutionsAreLocked!: boolean;
```

Comment:
`solutionsAreLocked` property should be properly decorated :thinking:. This is causing a false positive in `api/apps/api/test/legacy-project-import/legacy-project-import.e2e-spec.ts`, we are not providing a value for `solutionsAreLocked` (error) but API doesn't complain